docs/community/governance.mdx

```diff
@@ -110,6 +112,29 @@ All proposals must have a **sponsor** from the MCP steering group (maintainer, c
 
 Once proposals have a sponsor, they are assigned to the sponsor and are tagged `draft`.
 
+### SDK Versioning
+
+SDK maintainers have the freedom to do SDK version releases at their own pace. Each SDK version release **SHOULD** contain, as part of its code, the specification versions that are supported by the SDK version release.
+
+- SDK maintainers **MUST** use version release tags to mark compliance with different specification versions.
```

Comment:
Are we talking about release tags in GitHub or another avenue? Let's be explicit.